XTS/recovery/src/org/jboss/jbossts/xts/recovery/participant/ba/BAParticipantRecoveryModule.java

```diff
@@ -260,4 +265,4 @@ private void processParticipantsStatus()
     // Reference to the Object Store.
     private static RecoveryStore _recoveryStore = null ;
 
-}
\ No newline at end of file
+}
```

Comment:
Can you revert this? Thanks

Reply:
In the Intellij "Settings -> Editor -> General" section there is a checkbox that says "Ensure every saved file ends with a line break".